akka-docs/rst/java/futures.rst

```diff
@@ -260,3 +260,87 @@ After
 
 .. includecode:: code/docs/future/FutureDocTest.java
    :include: after
+
+Java 8, CompletionStage and CompletableFuture
+---------------------------------------------
+
+Starting with Akka 2.4.2 we have begun to introduce Java 8 ``java.util.concurrent.CompletionStage`` in Java APIs.
+It's a ``Future`` counterpart in Java; conversion from ``Future`` is done using ``scala-java8-compat`` library.
```

Comment:
This needs to say scala.concurrent.Future, otherwise Java people will think we mean java.concurrent.Future, which we do not mean here